totalopenstation/formats/carlson_rw5.py

```diff
@@ -19,7 +19,10 @@
 # along with Total Open Station.  If not, see
 # <http://www.gnu.org/licenses/>.
 
+import logging
+
 from totalopenstation.formats.conversion import horizontal_to_slope
+from .. import geo_to_debug
```

Comment:
I prefer to use absolute path here

Reply:
Ok, I had some trouble with path.
Sometimes it is correctly handle, sometimes not !

Reply:
Function removed but keep in mind the use of absolute path.